### PR TITLE
Fix issue with serializing from parquet to duckdb with timestamps

### PR DESCRIPTION
**Pull request by RobertLD**
- also add unit test to promote more tests be written around conversion (will add more later)

- [ ] Tests pass
- [x] ruff format
- [x] README.md updated (if relevant)
- [x] CHANGELOG.md entry added

### CODE REVIEW

**Comment by simonharrer** [comment]
See https://datacontract.com/#data-types which defines timestamp a synonym to timestamp_tz

**Comment by RobertLD** [comment]
I think the crux of the issue is that duckdb defines timestamp and datetime
as synonyms. Differing from that definition. With convert to duckdb type
only defining a one-to-one mapping; how would be support the equivalent
aliases

File: datacontract/export/sql_type_converter.py [hunk 1]
```diff
@@ -226,9 +226,10 @@ def convert_to_duckdb(field: Field) -> None | str:
         "bigint": "BIGINT",
         "date": "DATE",
         "time": "TIME",
-        "timestamp": "TIMESTAMP WITH TIME ZONE",
+        "datetime": "TIMESTAMP",
+        "timestamp": "TIMESTAMP",
```
Comment (jochenchrist) [inline datacontract/export/sql_type_converter.py:230]: timestamp is a synonym with timestamp_tz and should have the same type

Comment (jochenchrist) [reply datacontract/export/sql_type_converter.py:230]: plus `datetime` is not a valid Data Contract Data Typehttps://datacontract.com/#data-types

**Comment by jochenchrist** [comment]
I made some adjustments to merge this PR.

**Pull request merged**